app/views/conferences/_conference_details.html.haml

```diff
@@ -38,6 +38,10 @@
               = link_to "My Proposals", conference_program_proposals_path(conference.short_title), class: 'btn btn-default'
             - elsif can? :new, conference.program.events.new
               = link_to "Submit Proposal", new_conference_program_proposal_path(conference.short_title), class: 'btn btn-default'
+            - if current_user && current_user.booths.where(conference_id: conference.id).count > 0
+              = link_to 'My Booth Requests', conference_booths_path(conference.short_title), class: 'btn btn-default'
+            - elsif can? :new, conference.booths.new
+              = link_to 'Request Booth', new_conference_booth_path(conference.short_title), class: 'btn btn-default'
```

Comment:
It seems this always shows up, regardless if there is an active call for booths or not.